frontend/src/components/input/ListInput.vue

```diff
@@ -0,0 +1,145 @@
+    localModelValue.value = localModelValue.value.sort((a: string, b: string) =>
+      a.localeCompare(b),
+    )
+  }
+  emit("update:modelValue", [...new Set(localModelValue.value)] as string[]) //remove duplicates
+  if (!!localModelValue.value?.length) editMode.value = false
+  sortAlphabetically.value = false
+}
+const cancelEdit = () => {
+  if (!!localModelValue.value?.length) editMode.value = false
+}
+
+const toggleEditMode = () => {
+  editMode.value = !editMode.value
+}
+
+const adjustTextareaHeight = (textarea: HTMLTextAreaElement | null) => {
+  if (textarea) {
+    textarea.style.height = "auto" // Reset height first to recalculate based on content
+    textarea.style.height = `${textarea.scrollHeight}px ` // Set the height to match content
+  }
+}
+
+// Watch the `textAreaInput` value to adjust the height when content changes
+watch(textAreaInput, async () => {
+  await nextTick() // Wait for DOM render
+  const textarea = document.querySelector("textarea")
+  adjustTextareaHeight(textarea as HTMLTextAreaElement)
+})
+
+onMounted(() => {
+  // When data, show display mode. When empty, show edit mode
+  editMode.value = !props.modelValue.length
+})
+</script>
+
+<template>
+  <div>
+    <!-- Edit mode -->
+    <div v-if="editMode" class="flex flex-col gap-24">
+      <div class="flex flex-col gap-8">
+        <label id="list-input" class="ds-label-02-reg mb-4"> {{ label }}</label>
+        <textarea
+          id="list-input"
+          v-model="textAreaInput"
+          :aria-label="`${label} Input`"
+          class="ds-input h-auto resize-none overflow-hidden p-20"
+          placeholder="Geben Sie jeden Wert in eigene Zeile ein"
+          :rows="modelValue.length"
+          @input="adjustTextareaHeight($event.target as HTMLTextAreaElement)"
+        ></textarea>
+      </div>
+      <InputField
+        id="sortAlphabetically"
+        label="Alphabetisch sortieren"
+        :label-position="LabelPosition.RIGHT"
+      >
+        <Checkbox
+          id="sortAlphabetically"
+          v-model="sortAlphabetically"
+          aria-label="Alphabetisch sortieren"
+          class="ds-checkbox-mini bg-white"
+        />
+      </InputField>
+      <div class="flex w-full flex-row">
+        <div class="flex gap-16">
+          <TextButton
+            :aria-label="`${label} übernehmen`"
+            button-type="primary"
+            label="Übernehmen"
+            size="small"
+            @click.stop="addList"
+          />
+          <TextButton
+            aria-label="Abbrechen"
+            button-type="ghost"
+            label="Abbrechen"
+            size="small"
+            @click.stop="cancelEdit"
+          />
+        </div>
+      </div>
+    </div>
+    <!-- Display mode -->
+    <div v-else class="flex flex-col gap-16">
+      <div class="flex flex-col gap-8">
+        <span class="ds-label-02-reg mb-4">{{ label }}</span>
+        <ul class="m-0 flex flex-row gap-8 p-0">
```

Comment:
```suggestion
        <ul class="m-0 flex flex-row p-0">
```
Same as in edit mode, the design has only 4px from the label's `mb`